src/presentation/http/router/join.test.ts

```diff
@@ -7,7 +7,19 @@ describe('Join API', () => {
       const userId = 2;
       const accessToken = global.auth(userId);
 
-      const response = await global.api?.fakeRequest({
+      let response = await global.api?.fakeRequest({
+        method: 'POST',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: `/join/${hash}`,
+      });
+
+      // check if we added user to team
+      expect(response?.statusCode).toBe(200);
```

Comment:
do not check for the status code, check for actual behaviour instead

Reply:
i guess, that we already have test for checking actual behaviour so statusCode is sufficient condition, should i change it?

Reply:
Anyway `code` is not a guarantee that logic works fine. Somebody could break the logic, but leave 200 response. Tests should fail if someone will change response body of this method.

It's better to check response body for team member data, for example, `userId`

Reply:
done